x-pack/legacy/plugins/uptime/public/components/functional/monitor_status_details/monitor_status_bar/monitor_status_bar.tsx

```diff
@@ -52,7 +52,7 @@ export const MonitorStatusBarComponent: React.FC<Props> = ({
   const full = monitorStatus?.url?.full ?? '';
 
   return (
-    <EuiFlexGroup direction="column" gutterSize="none">
+    <EuiFlexGroup direction="column" gutterSize="none" responsive={false}>
```

Comment:
Why do we need all these `EuiFlexGroup` and `EuiFlexItem` elements? Maybe we could remove all of them and this is just text grouped in a `div`?

Reply:
I think we use flex group, because it makes sense in most cases, provides you lots of flexibility in terms of layout. Also provides you default gutter etc. 
underlying it doesn't make a diff, that is also a div.